ProjectSidewalk/SidewalkWebpage
# 3221 refactor gallery modal naming

Resolves #3221

Refactored Gallery code to replace 'Modal' with 'ExpandedView'.

##### Testing instructions
1. Start local instance
2. Checked functionality and UI of 'Gallery' to ensure consistency before and after changes.

##### Things to check before submitting the PR
